index.html

```diff
@@ -0,0 +1,249 @@
+      you see and have a project you need coded, don’t hesitate to contact me.
+    </p>
+    <div class="social">
+      <h3 class="connect">LET'S CONNECT</h3>
+      <ul class="social-links">
+         <li>
+          <a href="https://github.com/abenisamuel11" target="_blank" rel="noopener noreferrer"><img
+              src="./image/Github.png" alt="github logo"></a>
+        </li>
+       <li>
+          <a href="https://www.linkedin.com/in/abenezer-samuel/" target="_blank" rel="noopener noreferrer"><img
+              src="./image/linkdin.png" alt="linkdin logo" /></a>
+        </li>
+        <li>
+          <a href="" target="_blank" rel="noopener noreferrer"><img src="./image/A.png" alt="angle list icon" /></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/" target="_blank" rel="noopener noreferrer"><img src="./image/twitter.png"
+              alt="twitter.icon" /></a>
+        </li>
+
+        <li>
+          <a href="" target="_blank" rel="noopener noreferrer"><img src="./image/medium.png" alt="medium logo" /></a>
+        </li>
+       
+
+        
+      </ul>
+    </div>
+    <br />
+  </section>
+  </section>
+  
+  <section class="project-container">
+    <section class="work">
+      <img
+        src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl_4x.png?compress=1&resize=400x300&vertical=top" />
+      <h3 class="work-title">Tonic</h3>
+      <div class="profession">
+        <p>CANOPY</p>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>Back End Dev</p>
+        </div>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>2015</p>
+        </div>
+      </div>
+      <h3 class="description">
+        A daily selection of privately personalized reads;no accounts or
+        sign-ups required
+      </h3>
+
+      <ul class="languages">
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
+      </ul>
+
+      <button class="btn see-project">See project</button>
+    </section>
+
+    <section class="work">
+      <img src="./image/work2.png" />
+      <h3 class="work-title">MUlti-Post Stories</h3>
+      <div class="profession">
+        <p>CANOPY</p>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>Back End Dev</p>
+        </div>
+        <div class="expert">
+          <div class="bullet"></div>
+          <p>2015</p>
+        </div>
+      </div>
+      <h3 class="description">
+        A daily selection of privately personalized reads;no accounts or
+        sign-ups required
+      </h3>
+
+      <ul class="languages">
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JavaScript</li>
+      </ul>
+
+      <button class="btn see-project">See project</button>
+    </section>
+    <section class="work">
+      <img src="./image/work3.png" />
+      <h3 class="work-title">Tonic</h3>
+      <div class="profession">
```

Comment:
- [ ] please kindly add the font size and weight of the card title(`tonic`), tech languages and the see project text in the buttons so they match the original design.
original
![Screenshot (1207)](https://github.com/abenisamuel11/mobile-version-finish-new-season2/assets/65298048/ca48dc29-cc9d-4d30-a223-c91fdc3c3a8d)
yours
![Screenshot (1206)](https://github.com/abenisamuel11/mobile-version-finish-new-season2/assets/65298048/bb99832a-4d47-426c-9fed-36e319a6f720)